cultivation/scripts/running/run_performance_analysis.py

```
@@ -160,38 +374,176 @@ def main():
+        ("sleep_consistency_whoop", "Sleep Consistency (Whoop)", "%", None),
+        ("sleep_disturbances_per_hour_whoop", "Sleep Disturbances/hr (Whoop)", "", None),
+        ("strain_score_whoop", "Strain Score (Whoop)", "", None),
+        ("skin_temp_whoop", "Skin Temp (Whoop)", "°F", None),
+        ("resp_rate_whoop", "Resp Rate (Whoop)", "rpm", None),
+        ("steps_garmin", "Steps (Garmin)", "", None),
+        ("total_activity_garmin", "Total Activity (Garmin)", "min", lambda x: x/60 if x is not None else None),
+        ("resp_rate_garmin", "Resp Rate (Garmin)", "rpm", None),
+        ("vo2max_garmin", "VO2max (Garmin)", "ml/kg/min", None),
+    ]
+    if wellness_context:
+        summary_lines.append("\n--- Pre-Run Wellness Context (Data for {}) ---".format(run_date))
```

Ruff (0.8.2)

Undefined name `run_date`

(F821)